src/style.scss

```diff
@@ -0,0 +1,143 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+$--blue-color: #00acdc;
+$--gray-color: #616070;
+$--black-color: #060b35;
+$--white-color: #fff;
+$--standard-width: 166px;
+$--bold-font-weight: 700;
+$--medium-font-weight: 500;
+$--regular-font-weight: 400;
+
+body {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 406px;
+  padding: 16px;
+  margin: 20px auto;
+
+  &__image {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin: 17px 3px 20px;
+  }
+
+  &__name {
+    width: $--standard-width;
+    font-weight: $--medium-font-weight;
+    color: $--black-color;
+    font-size: 12px;
+    margin: 20px 0 4px;
+    line-height: 18px;
+    align-self: center;
+  }
+
+  &__code {
+    font-weight: $--regular-font-weight;
+    width: $--standard-width;
+    color: $--gray-color;
+    margin: 0 0 16px;
+    font-size: 10px;
+    line-height: 14px;
+    align-self: center;
+    text-align: left;
+  }
+
+  &__ratings {
+    width: $--standard-width;
+    display: flex;
+    align-self: center;
+    align-items: center;
+    justify-content: space-between;
+    height: 16px;
+    margin-bottom: 24px;
+  }
+
+  &__stars-4 {
```

Comment:
Consider using full styling from the previous task

Reply:
is it neccesary to take "copied-code" from the previous? Because i cant connect it with my already made code.